relay-profiling/src/client_sdk.rs

```diff
@@ -0,0 +1,7 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq)]
+pub struct ClientSdk {
```

Comment:
This already exists for `v2::ProfileMetadata`. I think I am in favour of just copy pasting it into every module where it is used, since technically this is part of the versioned protocol and not versioned separately.